idflakies-core/src/main/java/edu/illinois/cs/dt/tools/runner/RunnerPathManager.java

```diff
@@ -0,0 +1,69 @@
+package edu.illinois.cs.dt.tools.runner;
+
+import com.google.gson.Gson;
+import com.reedoei.eunomia.io.files.FileUtil;
+import edu.illinois.cs.dt.tools.utility.PathManager;
+import edu.illinois.cs.testrunner.data.results.TestRunResult;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
+public class RunnerPathManager {
+    public static final Path TEST_RUNS = Paths.get("test-runs");
+
+    public static Path testRuns(final File baseDir) {
```

Comment:
Should be able to remove baseDir, and then propagate back towards all uses of these methods to remove baseDir as well.